packages/odyssey-react/src/components/FieldGroup/FieldGroup.module.scss

```diff
@@ -33,12 +33,12 @@
   margin-inline: 0;
   padding-block: 0;
   padding-inline: 0;
-  color: $text-heading;
-  font-size: $size-title-4;
+  color: var(--FontColorTitle);
```

Comment:
You can drop the StyleDictionary value from the end here. Most folks have been calling them `Font/TextColor`, `FontSize`, and `FontLineHeight` since the values may map to something else after being themed.